retrying.py

```diff
@@ -63,13 +66,17 @@ def __init__(self,
                  wait_random_min=None, wait_random_max=None,
                  wait_incrementing_start=None, wait_incrementing_increment=None,
                  wait_exponential_multiplier=None, wait_exponential_max=None,
+                 wait_exp_full_jitter_base=None, wait_exp_full_jitter_max=None,
                  retry_on_exception=None,
                  retry_on_result=None,
                  wrap_exception=False,
                  stop_func=None,
                  wait_func=None,
                  wait_jitter_max=None):
 
+        # some vars are a tad bit long, and this is dry-er
+        default_if_none = lambda x, default: default if x is None else x
```

Comment:
Would be better to define a function at the top level module than compiling a lambda on each call to retry()